cmd/dlt/accountroles/cmd.go

```diff
@@ -229,11 +229,11 @@ func deleteAccountRoles(r *rosa.Runtime, env string, prefix string, clusters []*
 		r.Reporter.Infof(fmt.Sprintf("Successfully deleted the %saccount roles", roleTypeString))
 	case interactive.ModeManual:
 		r.OCMClient.LogEvent("ROSADeleteAccountRoleModeManual", nil)
-		policyMap, err := r.AWSClient.GetAccountRolePolicies(finalRoleList, prefix)
+		policies, excludedPolicies, err := r.AWSClient.GetAccountRolePolicies(finalRoleList, prefix)
```

Comment:
Would it be better to align the variable names as what was done with delete operator roles?
https://github.com/openshift/rosa/pull/2037/files#diff-a7b239b718526b39104909fe028dfe41eaf6f7e6210f739e11b58dd45494ef11R254

WDYT? Would you kindly advise if there was a reason to differentiate naming used between 
excludedPolicies and arbitraryPolicies?

Reply:
@davidleerh 
thanks for pointing it out, that inconsistence is not desired. For the naming, in pkg, in the common function to get policies with tag filter, I use excludedPolicies for common scenarios, but in cmd of account/operator-role, I use arbitrarypolicies for specific usage
Please help to re-reivew the pr.